### PR TITLE
Add Alertify.js

### PR DESCRIPTION
**Pull request by pczern**

### CODE REVIEW

**Comment by gtomitsuka** [comment]
I think that using Foundation is good enough. Could you send a screenshot with Bootstrap? I think we should stick with Foundation.

**Comment by pczern** [comment]
teenage programmers chatroom 1

**Comment by pczern** [comment]
It's very the same design without a login page

File: views/room.ejs [hunk 1]
```diff
@@ -135,37 +380,9 @@
...
-                var returnedUsername = prompt("Your current username is " + currentUser + ".\nType in a new username:");
-                if(!verifyEmptyness(returnedUsername) && returnedUsername !== null){
-                    localStorage.setItem('MTChat_Username', returnedUsername);
-                    currentUser = returnedUsername;
-                    $(".username").html(nameMessageHTMLPatternBegin + currentUser);
-                    socket.emit('postName', currentUser);
-                }
-            }
-            //Thanks to Jano González from SE for this
-            var verifyEmptyness = function(str) {
```
Comment (gtomitsuka) [inline views/room.ejs:164]: Why were these functions deleted?

Comment (pczern) [inline views/room.ejs:164]: It isn't, I changed the function on an other place. verifyEmptyness is also not deleted

**Pull request merged**